### PR TITLE
Hao tran

### PR DESCRIPTION
**Pull request by nohbdies**
test pull

### CODE REVIEW

File: routes/promotionRouter.spec.js [hunk 1]
```diff
@@ -1,89 +1,72 @@
-const supertest = require('supertest');
-const router = require('./promotionRouter.js');
+const supertest = require("supertest");
+const router = require("./promotionRouter.js");
 
-describe('promotion router', () => {
```
Comment (nmylynh) [inline routes/promotionRouter.spec.js:4]: hmm. you changed the formatting. are you using a different code formatter?

File: routes/promotionRouter.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
+const express = require("express");
```
Comment (nmylynh) [inline routes/promotionRouter.js:1]: formatting is good here

**Review by nmylynh — CHANGES_REQUESTED** [review]
Great job so far! A few things on formatting:

An easier way to just make sure your code is formatted correctly is through extensions.

**Importance of formatting:**  

- The reason why it's imperative to have a standard format is because of readability. Engineers are going to sit through hours of code everyday and having a standardized format of code will lessen the eye strain. 
- Usually repositories will have [a jenkins job](https://medium.com/slalom-build/automatically-generating-jenkins-jobs-d30d4b0a2b49#:~:text=Jenkins%20Job%20Builder%20is%20a,instance%20and%20its%20respective%20credentials.) that will check for formatting. If it isn't formatted right, your code will be rejected.

Lists of extensions to install:
1. ESlint
2. Prettier
3. Monokai Theme (optional, it's my preferred theme)
4. Rainbow Brackets (optional, but it helps clearly see nested functions)

I'd also like for you to request a PR on your own master branch. You can leave this one open, but i'll need you to request a merge to your own repository.

Added note: your origin is unknown. Are you using git through the CLI? You can [download github desktop](https://desktop.github.com/) to have a GUI of the git CLI. Try it out so that you can merge to your own repo.

File: routes/promotionRouter.spec.js (hunk 1 above)
Comment (nohbdies) [reply routes/promotionRouter.spec.js:4]: I haven't consciously done any formatting changes but I do have Prettier on VS code

**Pull request closed without merging**